test/CMakeLists.txt

```diff
@@ -10,5 +10,10 @@ else()
   target_link_libraries(s3select_test gtest gtest_main boost_date_time boost_thread boost_system pthread)
 endif()
 
+add_executable(s3select_perf_test s3select_perf_test.cpp)
+target_include_directories(s3select_perf_test PUBLIC ../include)
+target_link_libraries(s3select_perf_test gtest gtest_main boost_date_time boost_thread boost_system pthread)
```

Comment:
it failed the build
missing the arrow parquet libraries

Reply:
added the libraries